dev/fattest.simplicity/src/componenttest/rules/repeater/RepeatActions.java

```diff
@@ -80,7 +122,7 @@ public static RepeatTests repeat(String server, TestMode otherFeatureSetsTestMod
      * @param  testMode       The test mode to run the FeatureSet
      * @return                A FeatureReplacementAction instance
      */
-    public static FeatureReplacementAction forFeatureSet(Set<FeatureSet> allFeatureSets, FeatureSet featureSet, String server, TestMode testMode) {
+    public static FeatureReplacementAction forFeatureSet(Collection<FeatureSet> allFeatureSets, FeatureSet featureSet, String server, TestMode testMode) {
```

Comment:
Should this be List?

Reply:
For consistency yes but for this method the order does not matter.